src/pages/_layout.svelte

```diff
@@ -71,6 +71,8 @@
     overflow: hidden;
     max-width: 155rem;
     margin: 0 auto;
+    display: flex;
+    flex-direction: column;
```

Comment:
Making every page flex can have a lot of unintended consequences, can we just add a flex container on the 404 page itself? See our conversation on Slack.